### PR TITLE
BETA: Register mcreed.is-a.dev

### PR DESCRIPTION
**Pull request by M-creed**
Added `mcreed.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/mcreed.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "TXT": "4af6cbe09d82534a3e487fc8e6b54e"
```
Comment (Stef-00012) [inline domains/mcreed.json:7]: Is this a verification for some app?

Comment (M-creed) [reply domains/mcreed.json:7]: this is pages domain dns verification.
TXT record:
hostname: _github-pages-challenge-M-creed.mcreed.is-a.dev

value: 4af6cbe09d82534a3e487fc8e6b54e

Comment (CuteDog5695) [reply domains/mcreed.json:7]: > this is pages domain dns verification. TXT record: hostname: _github-pages-challenge-M-creed.mcreed.is-a.dev
> 
> value: 4af6cbe09d82534a3e487fc8e6b54e

then you'd have to make _github-pages-challenge-M-creed.mcreed.json for that then

Comment (M-creed) [reply domains/mcreed.json:7]: do you mean that i make `_github-pages-challenge-M-creed.mcreed.json` in every repo i need to add this domain to it ?

Comment (CuteDog5695) [reply domains/mcreed.json:7]: you'd need to make that for github pages verification

**Review by Stef-DP — APPROVED** [review]
Seems good now

**Review by iostpa — APPROVED** [review]
seems good to me now, ill merge it now

**Comment by M-creed** [comment]
> seems good to me now, ill merge it now

when i add the domain to repo page and check DNS for domain it's show me this message : 

> "mcreed.is-a.dev is improperly configured
> Your site's DNS settings are using a custom subdomain, mcreed.is-a.dev, that is set up as an A record. We recommend you change this to a CNAME record pointing to m-creed.github.io. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidARecordError)."

**Pull request merged**